rerun_py/rerun_sdk/rerun/_rerun2/datatypes/rotation_axis_angle_ext.py

```diff
@@ -16,3 +18,21 @@ def angle__field_converter_override(x: AngleLike) -> Angle:
             return x
         else:
             return Angle(rad=x)
+
+    @staticmethod
+    def native_to_pa_array_override(data: RotationAxisAngleArrayLike, data_type: pa.DataType) -> pa.Array:
+        from . import AngleArray, RotationAxisAngle, Vec3DArray
+
+        if isinstance(data, RotationAxisAngle):
+            data = [data]
+
+        axis_pa_array = Vec3DArray._native_to_pa_array([rotation.axis for rotation in data], data_type["axis"].type)
```

Comment:
I assume we use `_native_to_pa_array` here instead of `from_similar` to avoid the `wrap_array` extension wrapper?

Reply:
Hm I fear I got confused about these things actually.
In any case, calling `from_similar` gives me `ArrowNotImplementedError` because of `wrap_array` if I'm not mistaken

Reply:
okay, I minced different issues here. Please ignore my earlier comments on this. The answer to your question is a simple 'yes'. If we were to call `from_similar` we'd have the extension type wrapped in here causing errors like
```
pyarrow.lib.ArrowInvalid: Struct child array #1 does not match type field: extension<rerun.datatypes.Rotation3D<Rotation3DType>> vs dense_union<_null_markers: null=0, Quaternion: fixed_size_list<item: float not null>[4] not null=1, AxisAngle: struct<axis: fixed_size_list<item: float not null>[3] not null, angle: dense_union<_null_markers: null=0, Radians: float not null=1, Degrees: float not null=2> not null> not null=2>
```